rhel9/nvidia-driver

```diff
@@ -17,9 +17,7 @@ USE_HOST_MOFED="${USE_HOST_MOFED:-false}"
 DNF_RELEASEVER=${DNF_RELEASEVER:-""}
 RHEL_VERSION=${RHEL_VERSION:-""}
 RHEL_MAJOR_VERSION=9
-
```

Comment:
I don't see any changes to the `Dockerfile` for rhel9.

Reply:
Please check now